src/main/java/duke/util/Ui.java

```diff
@@ -0,0 +1,130 @@
+package duke.util;
+
+import duke.task.Task;
+
+/**
+ * Represents user interface of the chatbot.
+ */
+public class Ui {
+    private static final String HORIZONTAL_LINE = "____________________________________________________________\n";
+    private String message;
+
+    /**
+     * Constructor for UI class.
+     */
+    public Ui() {
+        this.message = "";
+    }
+
+    /**
+     * Prints the entry message of the chatbot.
+     */
+    public String printEntryMessage() {
+        String entryMessage = HORIZONTAL_LINE
+                + "Hello! I'm Chad \n"
+                + "What can I do for you? \n"
+                + HORIZONTAL_LINE;
+        return entryMessage;
+    }
+
+    /**
+     * Prints the exit message of the chatbot.
+     */
+    public void addExitMessage() {
+        this.message = "Bye. Hope to see you again soon!\n";
+    }
+
+    /**
+     * Prints the current message in the Ui object and clears the message.
+     */
+    public void printMessage() {
```

Comment:
Maybe can consider removing this method just for cleaner codebase since you're not using it anymore :) (lol could finally nitpick something)